images/gcb-docker-gcloud/Dockerfile

```diff
@@ -47,10 +47,12 @@ ENV HOME=/root
 RUN mkdir -p  $HOME/.docker/cli-plugins \
     && curl -fsSL "https://github.com/docker/buildx/releases/download/v0.10.2/buildx-v0.10.2.linux-amd64" --output $HOME/.docker/cli-plugins/docker-buildx \
     && chmod a+x $HOME/.docker/cli-plugins/docker-buildx
-# Link link /root/.docker to /builder/home/.docker because the default home for cloudbuild jobs is /builder/home
-# and this will make in uncessarry for each cloudbuild.yaml file in K8s project to specify HOME=/root
+# Link /root/.docker to /builder/home/.docker because the default home for cloudbuild jobs is /builder/home
+# Link /root/.docker/cli-plugins to /builder/home/.docker/cli-plugins
+# and this will make it unnecessary for each cloudbuild.yaml file in K8s project to specify HOME=/root
```

Comment:
let's install to the system-wide path instead of user paths?

https://github.com/docker/buildx#manual-download

Reply:
or https://github.com/docker/buildx#dockerfile

Reply:
+1 will test it out.

Reply:
Fixed.